frame/bridge/s2s/relay/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+authors     = ["Darwinia Network <hello@darwinia.network>"]
+description = "FRAME pallet to s2s relay"
+edition     = "2018"
+homepage    = "https://darwinia.network/"
+license     = "GPL-3.0"
+name        = "darwinia-s2s-relay"
```

Comment:
https://github.com/darwinia-network/parity-bridges-common/blob/master/modules/grandpa/src/lib.rs

I don't this this pallet is like relay, it is more like to be dispatch:

https://github.com/darwinia-network/parity-bridges-common/blob/master/modules/dispatch/src/lib.rs

Please fork Dispatch pallet or use Dispatch directly. There are all internal calls which can stay at some pallet else?

Reply:
this pallet do two things.
dispatch the relay-message and verify the message from remote chain.
So these two functions is more like a relay.

Reply:
Send message should use `pallet-bridge-messages`, right?


```
#[weight = T::WeightInfo::send_message_weight(payload)]		pub fn send_message(			origin,			lane_id: LaneId,			payload: T::OutboundPayload,			delivery_and_dispatch_fee: T::OutboundMessageFee,		) -&gt; DispatchResult {
```

https://github.com/darwinia-network/parity-bridges-common/blob/b9545dcb66da4ea572c9c43959cf088f2047a2c5/modules/messages/src/lib.rs#L296